tox.ini

```diff
@@ -15,6 +15,7 @@ deps =
     pytest==2.8.4
     pytest-cov==2.2.0
     pytest-django==2.9.1
+    isort==4.2.2
```

Comment:
Pff, well if they were, `pytest` would be below the other `pytest-` packages, an' I dun' like it!